diary-alexander-jaffray/modernized/18-ch-02-scotland.adoc

```diff
@@ -1,11 +1,12 @@
 == Chapter II.
 
-1664: Alexander Jaffray writes A Word of Exhortation, addressed to professors.
+[.blurb]
```

Comment:
I wasn't sure if this should be a [.section-summary-preface], but in the style guide it said that should come after a _subheading,_ so I made it a blurb instead. They looked really similar in the PDF preview.